src/patterns/molecules/breadcrumb/breadcrumb.hbs

```diff
@@ -0,0 +1,21 @@
+---
+name: Breadcrumb
+description: A type of navigation menu used to provide a list of links based on current page within the site hierarchy, allowing users to visit previous pages from the current page.
+order: 1
+notes: |
+    - Use the `.mzp-t-dark` options to use dark mode.
+    - Use a `<ol>` to use semantic HTML and structure.
+    - Using the `nav` element is optional, but reccomended.
+---
+<nav aria-label="breadcrumb">
+  <ol class="mzp-c-breadcrumb">
+    <li class="mzp-c-breadcrumb-item">
+      <a class="mzp-c-breadcrumb-item-link" href="#">Parent page</a>
```

Comment:
I wonder if we really need a class on each link. Maybe any link in an `mzp-c-breadcrumb-item` gives us enough description and specificity?

Reply:
Yeah I think you're right.